### ISSUE TITLE
Issue while converting from ONNX to TensorRT Engine

### ISSUE THREAD

**Issue by Darshcg** [NVIDIA/TensorRT#812]
Hi,
I am Converting my Custom Pytorch Model to TRT Engine file. I am able to convert the model to Onnx, but I am facing an Issue while converting from Onnx to the Engine file.
I am getting the error as:
AttributeError: ‘NoneType’ object has no attribute ‘serialize’.
I am giving the correct shape as an input to the build engine function.
And for the Resnet50 model, I am able to convert successfully but for my custom model, I am facing the above issue.
Please suggest if there is any specific reason behind that.
![Screenshot from 2020-09-25 15-13-55](https://user-images.githubusercontent.com/48723743/94932988-6ac23100-04e7-11eb-8062-221b651215aa.png)

Thanks

**Comment by Darshcg** [comment]
I tried with trtexec command, I am getting the error as attached below:

While parsing node number 195 [Resize]:
ERROR: ModelImporter.cpp:124 In function parseGraph:
[5] Assertion failed: ctx->tensors().count(inputName)
[09/25/2020-11:02:52] [E] Failed to parse onnx file
[09/25/2020-11:02:52] [E] Parsing model failed
[09/25/2020-11:02:52] [E] Engine creation failed
[09/25/2020-11:02:52] [E] Engine set up failed
&&&& FAILED TensorRT.trtexec # trtexec --onnx=last.onnx --batch=1 --saveEngine=last.engine

The command I used: trtexec --onnx=last.onnx --batch=1 --saveEngine=last.engine

**Comment by justinblaber** [comment]
One thing that might help is to look at the terminal output when you convert in the jupyter notebook. There's probably an unsupported operation being converted (for me, upsampling caused issues...)

**Comment by Darshcg** [comment]
Dear @justinblaber 

Thanks for your kind response.
I followed your advice(Looking at the terminal output when I am converting in a Notebook).
I got the output in the terminal as: 
**[TensorRT] ERROR: Network must have at least one output
[TensorRT] ERROR: Network validation failed.**

**Comment by ttyio** [comment]
Hello @Darshcg , thanks for reporting.

Could you check you onnx model using some visualization tool to see if the model really has no `output`? 

If you see the model has `output` but `onnx-trt` missed it, could you share your onnx file with us for better triage? thanks!

**Comment by Darshcg** [comment]
Hi @ttyio,

Here is the ONNX Model I am trying to convert: https://drive.google.com/file/d/1-XGtDAPrd5v-IQPRS3M44lssmjnkN1WR/view?usp=sharing.

**Comment by Darshcg** [comment]
Also, I tried with trtexec utility with the command:  **!trtexec --onnx=last.onnx --verbose --explicitBatch --shapes=input.1:1x3x320x192**



I get the log like this:
```
While parsing node number 195 [Resize -> "348"]:
--- Begin node ---
input: "338"
input: "340"
input: "340"
input: "347"
output: "348"
op_type: "Resize"
attribute {
  name: "coordinate_transformation_mode"
  s: "asymmetric"
  type: STRING
}
attribute {
  name: "cubic_coeff_a"
  f: -0.75
  type: FLOAT
}
attribute {
  name: "mode"
  s: "nearest"
  type: STRING
}
attribute {
  name: "nearest_mode"
  s: "floor"
  type: STRING
}

--- End node ---
ERROR: ModelImporter.cpp:124 In function parseGraph:
[5] Assertion failed: ctx->tensors().count(inputName)
[10/15/2020-07:01:54] [E] Failed to parse onnx file
[10/15/2020-07:01:54] [E] Parsing model failed
[10/15/2020-07:01:54] [E] Engine creation failed
[10/15/2020-07:01:54] [E] Engine set up failed
&&&& FAILED TensorRT.trtexec # trtexec --onnx=last.onnx --verbose --explicitBatch --shapes=input.1:1x3x320x192

```

**Comment by ttyio** [comment]
Hello @Darshcg , from your log it seems that we hit some issue come from the onnx `resize` node, we had some fix for `resize` in the latest 7.2 release, What is your TRT version,  could you take a try on the 7.2 release? thanks!

**Comment by ttyio** [comment]
Close since no response for more than 2 weeks, please reopen if you have more questions, thanks!
